src/fu-usb-device.c

```diff
@@ -130,10 +130,8 @@ fu_usb_device_open (FuDevice *device, GError **error)
 		if (idx != 0x00) {
 			g_autofree gchar *tmp = NULL;
 			tmp = g_usb_device_get_string_descriptor (priv->usb_device,
-								  idx, error);
-			if (tmp == NULL)
-				return FALSE;
-			fu_device_set_vendor (device, g_strchomp (tmp));
+								  idx, NULL);
+			fu_device_set_vendor (device, tmp ? g_strchomp (tmp) : "");
```

Comment:
I don't think setting "" makes sense. Maybe:

    if (tmp != NULL)
        fu_device_set_vendor (device, g_strchomp (tmp));

?

Reply:
I added that empty string so that the same string loading operation is not attempted any more, but well, I guess that would not harm either, maybe it succeeds on a new attempt later on (not in the case of the DW5821e/eSIM, where it always fails consistently). Let me update the commit.